src/pages/Host/Host.js

```diff
@@ -1,7 +1,153 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import variables from '../../styles/variables';
 
-const Host = () => {
-  return <div>Host</div>;
-};
+function Host() {
+  return (
+    <HostStyle.Container>
+      <HostStyle.Image>
+        <h1>호스팅을 시작해보세요.</h1>
+      </HostStyle.Image>
+      <HostStyle.SectionContainer>
+        <HostStyle.SectionIntro>
+          <Link to="/">⇱ 나가기</Link>
+          <h1>호스트님 안녕하세요.</h1>
+        </HostStyle.SectionIntro>
+        <HostStyle.SectionInputWrap>
+          <label htmlFor="Name">Name</label>
+          <input type="text" name="Name" id="Name" />
+          <label htmlFor="Nick name">Nick name</label>
+          <input type="text" name="Nick name" id="Nick name" />
+          <label htmlFor="E-mail">E-mail</label>
+          <input type="text" name="E-mail" id="E-mail" />
+          <label htmlFor="Phone">Phone</label>
+          <input type="text" name="Phone" id="Phone" />
```

Comment:
아래의 코드와 같이 <label> 안에 <input> 을 중첩시킬 경우 연관이 암시적이므로 for 및 id속성이 필요없어 불필요한 속성 사용을 줄일 수 있습니다. 아래 코드 예시를 확인해주세요.
```suggestion
          <label>
               <input type="text" name="Name" />
                Name
          </label>
```

Reply:
아래 코드와 같이 수정했습니다.